File: android_samples_java/19.2 SukritiTest/app/src/main/java/com/sukriti/sukrititest/SecondActivity.java
package com.sukriti.sukrititest;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.AppCompatButton;
import androidx.appcompat.widget.AppCompatEditText;
import androidx.appcompat.widget.AppCompatImageView;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.LinearLayout;

public class SecondActivity extends AppCompatActivity implements CallbackInterface {

    LinearLayout firstcontainer, secondcontainer, thirdcontainer;
    OneFragment oneFragment;
    TwoFragment twoFragment;
    ThreeFragment threeFragment;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);

        firstcontainer = findViewById(R.id.firstcontainer);
        secondcontainer = findViewById(R.id.secondcontainer);
        thirdcontainer = findViewById(R.id.thirdcontainer);

        Bundle bundle = new Bundle();
        bundle.putInt("initvalue", Integer.parseInt(getIntent().getStringExtra("initvalue")));

        oneFragment = new OneFragment();
        twoFragment = new TwoFragment();
        threeFragment = new ThreeFragment();

        replaceFragment(oneFragment, firstcontainer, null, "OneFragment");
        replaceFragment(twoFragment, secondcontainer, bundle, "TwoFragment");
        replaceFragment(threeFragment, thirdcontainer, null, "ThreeFragment");
    }

    public void replaceFragment(Fragment fragment, LinearLayout ll, Bundle bundle, String tag) {
        if (bundle != null) {
            fragment.setArguments(bundle);
        }

        FragmentManager fm = getSupportFragmentManager();
        fm.beginTransaction().add(ll.getId(), fragment, tag).commit();
    }

    @Override
    public void increment() {
        twoFragment.incrementNumber();
    }

    @Override
    public void decrement() {
        twoFragment.decrementNumber();
    }

}

File: android_jetpack/SukritiTest/app/src/main/java/com/sukriti/sukrititest/SecondActivity.kt
package com.sukriti.sukrititest

import android.os.Bundle
import android.widget.LinearLayout
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import com.sukriti.sukrititest.databinding.ActivityFirstBinding
import com.sukriti.sukrititest.databinding.ActivitySecondBinding

class SecondActivity : AppCompatActivity(), CallbackInterface {
    var oneFragment: OneFragment? = null
    var twoFragment: TwoFragment? = null
    var threeFragment: ThreeFragment? = null

    lateinit var binding: ActivitySecondBinding;

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivitySecondBinding.inflate(layoutInflater)
        setContentView(binding.root)

        val bundle = Bundle()
        bundle.putInt("initvalue", getIntent().getStringExtra("initvalue")!!.toInt())

        oneFragment = OneFragment()
        twoFragment = TwoFragment()
        threeFragment = ThreeFragment()

        replaceFragment(oneFragment!!, binding.firstcontainer!!, null, "OneFragment")
        replaceFragment(twoFragment!!, binding.secondcontainer!!, bundle, "TwoFragment")
        replaceFragment(threeFragment!!, binding.thirdcontainer!!, null, "ThreeFragment")
    }

    fun replaceFragment(fragment: Fragment, ll: LinearLayout, bundle: Bundle?, tag: String?) {
        if (bundle != null) {
            fragment.setArguments(bundle)
        }

        val fm = getSupportFragmentManager()
        fm.beginTransaction().add(ll.getId(), fragment, tag).commit()
    }

    override fun increment() {
        twoFragment!!.incrementNumber()
    }

    override fun decrement() {
        twoFragment!!.decrementNumber()
    }
}